### DISCUSSION TITLE
Version 0.18 is available

### DISCUSSION THREAD

**Discussion by andsel** [moquette-io/moquette#890] [Announcements]
Today was tagged and released Moquette 0.18.0 https://github.com/moquette-io/moquette/releases/tag/v0.18.0, it's the first version with almost complete support for MQTT 5. The leftovers are some minor requirements and the enhanced authentication.

**Comment by ThisOtterBeGood** [comment]
Hi,
will 0.18 be made available in maven central? I cannot find it there.

**Reply by andsel** [reply]
No, for simplicity of management of the project we rely on Jitpack

**Reply by hylkevds** [reply]
If you need a version that is published on maven central, maybe because your own software is published there, you can use the packages we publish: 
https://central.sonatype.com/artifact/de.fraunhofer.iosb.io.moquette/moquette-broker
They also contain some PRs not merged into the official moquette yet.
